focal/3_international_travel_with_minor_and_upgrade_examples.html

```diff
@@ -4,15 +4,15 @@ <h3>Focal Use Case: International Travel with Minor and Upgrade</h3>
   <pre class="example nohighlight" title="Malathi's passport (simple model)">
 {
       "@context": [
-            "https://w3id.org/credentials/v1",
+            "https://www.w3.org/ns/credentials/v2",
             "https://example.com/travel-vocab/v1"
       ],
-      "id": "urn:uuid:9f6878c8-73c7-11e8-ab37-23a1a3504fd0",
+      "id": "did:example:9f6878c8-73c7-11e8-ab37-23a1a3504fd0",
```

Comment:
Should the VCs and the passport have a DID for an id?

Reply:
Why not? Paper passport has an ID...